i18n/md/collections/2023/en/conclusion.md

```diff
@@ -0,0 +1,22 @@
+---
+id: conclusion
+language: en
+---
+
+Let's recap some of the stories and dramas of 2023:
+
+- Svelte code base ditching TypeScript files (while preserving type safery)
+- Controversies around React (either too old, too complicated, too PHP-like...)
+- The fall of Rome and the rise of Biome
+- Angular rejuvenated
+- The release of Bun v1
+- Astro 4.0 released, introducing a new toolbar that makes the DX even better
+- Stylex open-sourced by Meta, at last!
+
+We saw the rise of AI tools that can generate UI from prompts and ever images making people wonder if there will be frontend developers in a few years.
```

Comment:
I am a little confused about the part of the sentence "ever images". Is this a type of image? If not, maybe better to remove "ever"?

Reply:
Sorry, it's a typo, I meant `even`, not `ever`!